test/iidm/util/NodeBreakerTopologyTest.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <boost/test/unit_test.hpp>
+
+#include <powsybl/iidm/BusbarSection.hpp>
+#include <powsybl/iidm/Load.hpp>
+#include <powsybl/iidm/LoadAdder.hpp>
+#include <powsybl/iidm/Network.hpp>
+#include <powsybl/iidm/VoltageLevel.hpp>
+#include <powsybl/iidm/util/NodeBreakerTopology.hpp>
+
+#include "../NetworkFactory.hpp"
```

Comment:
Is it really necessary to include with the relative path? Maybe it could be manage in the CMakeLists.